bundles/org.openhab.binding.tesla/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -4,6 +4,37 @@
 	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 
+	<channel-type id="ar_destination">
+		<item-type>String</item-type>
+		<label>Route destination</label>
+		<description>Name of the destination</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="ar_location" advanced="false">
+		<item-type>Location</item-type>
+		<label>Route location</label>
+		<description>Location of the destination</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="ar_distancetoarrival">
+		<item-type>Number</item-type>
+		<label>Distance to arrival</label>
+		<description>Distance to drive to the destination</description>
```

Comment:
Unit could be provided in the description (and README).

Reply:
I'm reopening this. Suggestion is to add "miles" to the description, so that it's known what to expect when linking a `Number` item (without unit).
```suggestion
		<description>Distance to drive to the destination (in miles)</description>
```